File: src/tutorial/standalone/pre.py
import sys

for n in range(1,101):
	fizzbuzz = False
	if n % 3 == 0:
		fizzbuzz = True
		sys.stdout.write('fizz')
	if n % 5 == 0:
		fizzbuzz = True
		sys.stdout.write('buzz')
	if not fizzbuzz:
		sys.stdout.write(str(n))
	sys.stdout.write('\n')

File: src/tutorial/standalone/pre.cpp
#include <iostream>
#include <cstdlib>

int main(){
	for(int i = 0; i < 101; ++i){
		bool fizzbuzz = false;
		if(i % 3 == 0){
			fizzbuzz = true;
			std::cout << "fizz";
		}
		if(i % 5 == 0){
			fizzbuzz = true;
			std::cout << "buzz";
		}
		if(!fizzbuzz){
			std::cout << i;
		}
		std::cout << '\n';
	}
	return EXIT_SUCCESS;
}

